kernel/.gitignore

```diff
@@ -1,2 +1,3 @@
 linux-*
 !*.patch
+!*.sha256
```

Comment:
Perhaps instead of adding negative rules, we could make the positive ones more restrictive, e.g. `kernel/linux-*`.

Reply:
That will still cause `kernel/linux-*.tar.xz.sha256` to be ignored, which is what I am trying to avoid.

Reply:
Bad example then, I thought the other files were in the root folder. Then: `kernel/linux-*.tar.xz`.

Reply:
The extracted folder (`linux-*`) needs to be ignored too, hence the initial exclusion. Realistically, this is the only other negative rule that we need for this scenario.